<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClipMergeWidget</class>
 <widget class="QWidget" name="ClipMergeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clip Merge</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="inputDirLabel">
     <property name="text">
      <string>1. Select input directory</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="inputDirEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="inputDirButton">
       <property name="text">
        <string>Select Directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="inputFilesLabel">
     <property name="text">
      <string>2. Select files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="inputFileView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="matchingInputFileButton">
       <property name="text">
        <string>Select All Files In Route</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearSelectedInputFilesButton">
       <property name="text">
        <string>Clear Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="outputDirLabel">
     <property name="text">
      <string>3. Select output file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="outputFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="outputFileButton">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="settingsLabel">
     <property name="text">
      <string>4. Output Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="videoEncodeComboBox"/>
     </item>
     <item>
      <widget class="QCheckBox" name="includeGpsCheckBox">
       <property name="text">
        <string>Include GPS Data</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="compressionLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Video Compression Factor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="compFactorSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>51</number>
       </property>
       <property name="value">
        <number>28</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="mergeButton">
     <property name="text">
      <string>Merge</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearSelectedInputFilesButton</sender>
   <signal>released()</signal>
   <receiver>inputFileView</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
